packages/react-keytips/src/hooks/useHotkeys.ts

```diff
@@ -54,7 +54,11 @@ const isKeyMatchingKeyboardEvent = (
   return false;
 };
 
-export const useHotkeys = (hotkeys: Hotkey[], target?: Document) => {
+export const useHotkeys = (
+  hotkeys: Hotkey[],
+  invokeEvent: 'keyup' | 'keydown' = 'keydown',
```

Comment:
nit: this type could potentially be reused for component properties and the hook.